### PR TITLE
Fixed exit conditions under query that do not release connections

### PR DESCRIPTION
**Pull request by sushshring**

### CODE REVIEW

File: src/services/database/svc/mysql-uow.service.ts [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import { Inject, Injectable } from 'injection-js';
 import { MysqlError, PoolConnection } from 'mysql';
 import { defer, from, Observable } from 'rxjs';
-import { catchError, mergeMap } from 'rxjs/operators';
+import { catchError, map, mergeMap } from 'rxjs/operators';
```
Comment (mcao) [inline src/services/database/svc/mysql-uow.service.ts]: Just curious, why is map being imported here? I don't see it being used anywhere in this file.

Comment (sushshring) [reply src/services/database/svc/mysql-uow.service.ts]: Bc I forgot to remove it. Good catcg

Comment (ThePBJain) [reply src/services/database/svc/mysql-uow.service.ts]: has this been fixed?

File: package.json [hunk 2]
```diff
@@ -30,7 +30,7 @@
     "extension": [
       ".ts"
     ],
-    "lines": 70,
+    "lines": 65,
```
Comment (ThePBJain) [inline package.json]: what is this for?

File: src/services/database/svc/mysql-uow.service.ts (hunk 1 above)
Comment (sushshring) [reply src/services/database/svc/mysql-uow.service.ts]: Fixing now.

File: package.json (hunk 2 above)
Comment (sushshring) [reply package.json]: Pass the travis build

File: src/services/database/svc/mysql-uow.service.ts (hunk 1 above)
Comment (sushshring) [reply src/services/database/svc/mysql-uow.service.ts]: Fixed

**Pull request merged**